ui/app/controllers/evaluations/index.js

```diff
@@ -5,6 +5,10 @@ import { tracked } from '@glimmer/tracking';
 export default class EvaluationsController extends Controller {
   queryParams = ['pageSize'];
 
+  get evaluations() {
+    return this.model;
+  }
```

Comment:
This is a reactive getter that I'm expecting to update every time model updates, and everything associated in `ListTable` should update when this getter changes. Still unclear why the changes aren't updating here.

cc:  @LevelbossMike